axon-server/administration/backup-and-messaging-only-nodes.md

```diff
@@ -1,59 +1,132 @@
-# Backup and Messaging only Nodes
+# Backup and Messaging-only Nodes
```

Comment:
Don't these file changes belong to [this](https://github.com/AxonIQ/reference-guide/pull/132) PR you've provided?